pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -359,10 +386,13 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 		q.Execute(testCtx, kubeTestsCopy, parallelism, status.Run)
 		tests = append(tests, kubeTestsCopy...)
 
-		// I thought about randomizing the order of the kube, storage, and openshift tests, but storage dominates our e2e runs, so it doesn't help much.
-		storageTestsCopy := copyTests(storageTests)
-		q.Execute(testCtx, storageTestsCopy, max(1, parallelism/2), status.Run) // storage tests only run at half the parallelism, so we can avoid cloud provider quota problems.
-		tests = append(tests, storageTestsCopy...)
+		// run storage tests only if storage capability enabled
+		if storageEnabled {
```

Comment:
With [this `nil`-ing][1] above, I don't understand why we need a conditional here.  Can we leave this block of code unchanged?

[1]: https://github.com/openshift/origin/pull/27470#discussion_r999661183

Reply:
Right
Fixed at 9e66b24935a39a9d8f6abbab925179cf4e28d4a1